### PR TITLE
file management and norminette friendly

### PR DESCRIPTION
**Pull request by gharazka**
norminette fixed but found some bugs while fixing it
I'll work on the builtins architecture now, push and then try to fix bugs while you can start builtins, should be done around midnight

### CODE REVIEW

**Comment by nuketayev** [comment]
what

**Comment by nuketayev** [comment]
just let me know when builtin architecture is on

File: inc/minishell.h [hunk 1]
```diff
@@ -25,7 +25,6 @@
 typedef enum e_token_type
 {
 	TOKEN_END,
-	TOKEN_COMMAND,
```
Comment (nuketayev) [inline inc/minishell.h:28]: Why?

File: inc/minishell.h [hunk 2]
```diff
@@ -41,32 +40,32 @@ typedef struct s_token
 	char			*value;
 }					t_token;
 
-extern volatile sig_atomic_t	g_sigint_received;
+//signal_handler.c
+void	set_handler_one(struct sigaction *sa);
+void	set_handler_two(struct sigaction *sa);
 
-typedef struct s_program
```
Comment (nuketayev) [inline inc/minishell.h:46]: what? Where it came from? I think i didnt see it before or im too tired

**Review by nuketayev — APPROVED** [review]

File: inc/minishell.h (hunk 1 above)
Comment (gharazka) [reply inc/minishell.h:28]: I have a plan, trust

**Comment by nuketayev** [comment]
you merge it, Its difficult to see changes when functions moved to other files. Looks alright to me overall

File: inc/minishell.h (hunk 2 above)
Comment (gharazka) [reply inc/minishell.h:46]: I copied everything from my pipex and didn't delete until now

**Pull request merged**